modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeSetResultsSetTotalMethodCheck.java

```diff
@@ -5,79 +5,173 @@
 		throws Exception {
 
-		if (!fileName.endsWith(".java") && !fileName.endsWith(".jsp") &&
-			!fileName.endsWith(".jspf")) {
+		if (fileName.endsWith(".java")) {
+			content = _formatJava(content, fileName);
+		}
+		else if (fileName.endsWith(".jsp") || fileName.endsWith(".jspf")) {
+			content = _formatContent(content, content, fileName);
+		}
 
-			return content;
+		return content;
+	}
+
+	@Override
+	protected String[] getValidExtensions() {
+		return new String[] {"java", "jsp", "jspf"};
+	}
+
+	private String _formatContent(
+		String content, String fileContent, String fileName) {
+
+		String newContent = fileContent;
+
+		Matcher matcher = _setResultsPattern.matcher(content);
+
+		String setResultMethodCall = null;
+
+		int setResultIndex = 0;
+
+		if (matcher.find()) {
+			setResultIndex = matcher.start();
+
+			setResultMethodCall = JavaSourceUtil.getMethodCall(
+				content, setResultIndex);
 		}
 
-		content = _removeSetTotal(content, fileName);
-		content = _replaceSetResults(content, fileName);
+		matcher = _setTotalPattern.matcher(content);
 
-		return content;
+		String setTotalMethodCall = null;
+
+		int setTotalIndex = 0;
+
+		if (matcher.find()) {
+			setTotalIndex = matcher.start();
+
+			setTotalMethodCall = JavaSourceUtil.getMethodCall(
+				content, setTotalIndex);
+		}
+
+		if (!hasClassOrVariableName(
+				"SearchContainer", content, newContent, fileName,
+				setResultMethodCall) &&
+			!hasClassOrVariableName(
+				"SearchContainer", content, newContent, fileName,
+				setTotalMethodCall)) {
+
+			return newContent;
+		}
+
+		return StringUtil.replace(
+			newContent, content,
+			_getNewMethodCall(
+				content, setResultIndex, setResultMethodCall, setTotalIndex,
+				setTotalMethodCall));
 	}
 
-	private String _removeSetTotal(String content, String fileName) {
-		String newContent = content;
+	private String _formatJava(String content, String fileName)
+		throws Exception {
 
-		Matcher setTotalMatcher = _setTotalPattern.matcher(content);
+		JavaClass javaClass = JavaClassParser.parseJavaClass(fileName, content);
 
-		while (setTotalMatcher.find()) {
-			if (hasClassOrVariableName(
-					"SearchContainer", content, content, fileName,
-					JavaSourceUtil.getMethodCall(
-						content, setTotalMatcher.start()))) {
+		if (javaClass == null) {
+			return content;
+		}
 
-				newContent = StringUtil.removeSubstring(
-					newContent, setTotalMatcher.group());
+		for (JavaTerm childJavaTerm : javaClass.getChildJavaTerms()) {
+			if (!childJavaTerm.isJavaMethod()) {
+				continue;
 			}
+
+			JavaMethod javaMethod = (JavaMethod)childJavaTerm;
+
+			String javaMethodContent = javaMethod.getContent();
+
+			content = _formatContent(javaMethodContent, content, fileName);
 		}
 
-		return newContent;
+		return content;
 	}
 
-	private String _replaceSetResults(String content, String fileName) {
-		String newContent = content;
+	private String _getNewMethodCall(
+		String javaMethodContent, int setResultIndex,
```

Comment:
@TamyrisTorres, can we rename `javaMethodContent` to `content`?

According to the calls to this method, the first parameter can hold, indeed, a Java Method, but it can also hold a content which isn't only a Java Method, in light of this would make more sense we name this parameter as `content` only. Does it make sense to you?